### PR TITLE
EZP-30397: Unify JavaScript event names

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30397
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

## New names

**Admin UI** (this PR)

| old name      | new name
| ------------- | ---
| invalidFileSize       | ez-invalid-file-size
| addressNotFound       | ez-address-not-found
| cancelErrors       | ez-cancel-errors

**Page builder** (https://github.com/ezsystems/ezplatform-page-builder/pull/334)

| old name      | new name
| ------------- | ---
| openUdw       | ez-open-udw
| openAirtimePopup       | ez-open-airtime-popup
| postUpdateBlocksPreview       | ez-post-update-blocks-preview
| pbIframeLoaded       | ez-page-builder-iframe-loaded
| pbHideTools       | ez-page-builder-hide-tools

**Form Builder** (https://github.com/ezsystems/ezplatform-form-builder/pull/156 + https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/180)

| old name      | new name
| ------------- | ---
| openUdw       | ez-open-udw
| updateFieldName       | ez-update-field-name
| fbFormBuilderLoaded       | ez-form-builder-loaded
| fbFormBuilderUnloaded       | ez-form-builder-unloaded

## Deleted event listener for:
- pbPreviewReloaded (https://github.com/ezsystems/ezplatform-page-builder/pull/334)

## Names that does not change

**Admin UI**

| name |   
| ------------- | 
| ez-notify       |
| ez-bookmark-change       |
| ez-draft-conflict-modal-hidden       |

**Admin UI Modules**

| name |   
| ------------- | 
| ez-content-tree-refresh       |

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js [hunk 1]
```diff
@@ -357,7 +357,7 @@
             {
                 isValueValidator: false,
                 selector: `${SELECTOR_FIELD} ${SELECTOR_ADDRESS_INPUT}`,
-                eventName: 'addressNotFound',
+                eventName: 'ez-address-not-found',
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js]: Can be a `const` maybe?

File: src/bundle/Resources/public/js/scripts/fieldType/ezimage.js [hunk 2]
```diff
@@ -43,7 +43,7 @@
             sizeContainer.title = fileSize;
 
             preview.querySelector('.ez-field-edit-preview__action--preview').href = URL.createObjectURL(files[0]);
-            this.fieldContainer.querySelector(SELECTOR_INPUT_ALT).dispatchEvent(new CustomEvent('cancelErrors'));
+            this.fieldContainer.querySelector(SELECTOR_INPUT_ALT).dispatchEvent(new CustomEvent('ez-cancel-errors'));
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/scripts/fieldType/ezimage.js]: `const` maybe?

File: src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js (hunk 1 above)
Comment (tischsoic) [reply src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js]: You mean, move it above class definition? :)

Comment (sunpietro) [reply src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js]: Right

File: src/bundle/Resources/public/js/scripts/fieldType/ezimage.js (hunk 2 above)
Comment (tischsoic) [reply src/bundle/Resources/public/js/scripts/fieldType/ezimage.js]: Moved to const: https://github.com/ezsystems/ezplatform-admin-ui/pull/956/commits/a5f8a64f1a22653049210ba9c7b1d95d9bd22e9b

File: src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js (hunk 1 above)
Comment (sunpietro) [reply src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js]: Still no `const`?

**Pull request merged**